screens/loginPage.js

```diff
@@ -52,7 +55,10 @@ class loginPage extends Component {
                     profile_picture: result.additionalUserInfo.profile.picture,
                     first_name: result.additionalUserInfo.profile.given_name,
                     last_name: result.additionalUserInfo.profile.family_name,
-                    created_at: Date.now()
+                    created_at: Date.now(),
+                    year: year,
```

Comment:
it might be better to name year graduation_year in the database.

Reply:
done